documentation/shared/attributes.adoc

```diff
@@ -33,6 +33,7 @@
 :JMXExporter: link:https://github.com/prometheus/jmx_exporter[JMX Exporter documentation^]
 :PrometheusHome: link:https://github.com/prometheus[Prometheus^]
 :PrometheusConfig: link:https://prometheus.io/docs/prometheus/latest/configuration/configuration[Configuration^]
+:SupportedPrometheusVersion: master
```

Comment:
Do we really need the variable if it is used only in two places next to each other? I think it might be better to keep the version in the example directly where it is more likely to be seen and noticed that it is wrong since we only rarely fo through the variable values and they seem to be often outdated.

Reply:
Yes, I was thinking that. As long as we keep the note for non-Kubernetes users, I'm happy to remove this variable and use `master` in the commands.